firmware/src/core/utils/HTTPClientWrapper.h

```diff
@@ -0,0 +1,57 @@
+#ifndef HTTP_CLIENT_WRAPPER_H
+#define HTTP_CLIENT_WRAPPER_H
+
+#include <Arduino.h>
+#include <HTTPClient.h>
```

Comment:
This `HTTPClient.h` does not have to be included here, it is an implementation detail of the `HTTPClientWrapper`

Reply:
not sure I understand...this is the implementation of HTTPClientWrapper.

Reply:
and who of the _users_ of the `HTTPClientWrapper` needs to know about the `HTTPClient` ?